<ui version="4.0" >
 <class>Boolval_UI</class>
 <widget class="QWidget" name="Boolval_UI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>259</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QGroupBox" name="title" >
       <property name="font" >
        <font>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="title" >
        <string>GroupBox</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignLeading</set>
       </property>
       <property name="flat" >
        <bool>false</bool>
       </property>
       <property name="checkable" >
        <bool>false</bool>
       </property>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QCheckBox" name="checkBox" >
          <property name="text" >
           <string>CheckBox</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
